menus/setup-identity.ts

```diff
@@ -34,7 +34,7 @@ export default async function ({ returnAssets = false } = {}): Promise<any> {
   await status('connecting to network...')
   await connect(network)
   const account = await loadAccount(mnemonic)
-  const keypairs = await getKeypairs(account, mnemonic)
+  const keypairs = await getKeypairs(mnemonic)
```

Comment:
The calls of this function need to be updated to remove "account" from the parameters. 

Files that I found that need update:

- `menus/setup-identity.ts`
- `menus/setup-claimer.ts`
- `menus/create-ctype.ts`

Reply:
Done thank you :)